api/config/config_test.json

```diff
@@ -3,7 +3,7 @@
   "firecloud": {
     "debugEndpoints": false,
     "billingAccountId": "014D91-FCB792-33D2C0",
-    "billingProjectPrefix": "aou-test-free1-",
+    "billingProjectPrefix": "aou-test-f1-",
     "registeredDomainName": "all-of-us-registered-test",
```

Comment:
Does this exist yet in FC prod?

Reply:
I created it, and added the affiliated Google Group to the ACL on the CDR version we're using.